build.gradle.kts

```diff
@@ -46,6 +46,10 @@ dependencies {
     testImplementation(Junit5.kotlinRunner)
     testImplementation(Json.library)
     testRuntimeOnly(Junit5.engine)
+
+    implementation(Prometheus.common)
+    implementation(Prometheus.hotspot)
```

Comment:
jeg _tror_ vi kun trenger `implementation(Prometheus.common)` her - de andre er for "scraping" og jvm metrikler

Reply:
Det stemmer nok :)